Header header
uint16 UNKOWN=0
uint16 SCAN=8706
uint16 OBJECT=8737
uint16 VEHICLE_STATE=10245
uint16 ERRORS_AND_WARNINGS=8240
uint16 COMMAND_INTERFACE=8208

uint16 type
uint64 ntp_time
ScanData scan_data
ObjectData object_data
